modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -426,3 +532,31 @@ const NewOverflowList = () => {
   );
 };
 ```
+
+---
+
+### Menu (Main)
+
+We've updated the depth value of `Menu.Card` from `1` to `3`. This is a visually breaking change.
+
+---
+
+### Modal
+
+We've updated the `Modal` component to include new touch and responsive behaviors. The responsive
+`Modal` will be displayed when users are browsing on screen sizes between `320px` and `768px` and
+will have the following visual differences compared to the standard `Modal`:
+
+- Padding between the edge of `Modal.Card` and its contents reduced to `16px` (standard: `32px`)
```

Comment:
I think the padding for the responsive `Modal` is supposed to be `16px` (that's what it is in the [code](https://github.com/Workday/canvas-kit/blob/prerelease/major/modules/react/modal/lib/ModalCard.tsx#L21)), but #1815 says it's supposed to be `24px` in the PR description:

![Screen Shot 2022-10-04 at 6 07 20 PM](https://user-images.githubusercontent.com/146020/193957790-0646b7e2-a14a-4e14-b269-b945cb27bff2.png)

Reply:
Assuming `16px` is correct since that's what's being used in the code.